### DISCUSSION TITLE
rename umap clusters

### DISCUSSION THREAD

**Discussion by yamihn** [GreenleafLab/ArchR#1577] [Questions / Documentation] [answered]
Hi! is it possible to change the name of the clusters in the umap??
![reclustering](https://user-images.githubusercontent.com/109409803/185792511-dd56f3e6-ee61-4ca0-a47d-e411933391ed.png)

I would to show in the umap the real name of clusters as the legend shows: 
C0, C1, C2, C3 C4 C5 C6 C7 C9 C10 C11 C12 C13 
not the numbers, because it gets me confused .
Thank you very much

**Comment by rcorces** [comment, marked as answer]
add a new column to `cellColData` with the names that you want. An easy way to do this is with the `mapLabels()` function
https://www.archrproject.com/reference/mapLabels.html
